lab-brian/test/greet-test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const greet = require('../lib/greet.js');
+const expect = require('chai').expect;
+
+describe('Greet Module', function(){
+  describe('#sayHello', function(){
+    it('should return a string of hey name', function(){
+      var result = greet.sayHello('brian');
```

Comment:
Good work storing the value of the method call in a variable to more cleanly reference it in your test below!